posthog/models/activity_logging/activity_log.py

```diff
@@ -238,7 +235,7 @@ def dict_changes_between(model_type: ActivityScope, previous: Dict[Any, Any], ne
 
 
 def log_activity(
-    organization_id: UUIDT,
+    organization_id: Union[str, UUIDT],
```

Comment:
When can this be a str? Why?

Reply:
We [expose the organization_id as a string here](https://github.com/PostHog/posthog/blob/master/posthog/api/routing.py#L83). I'll uuid-ify here and check if that method shouldn't expose a string separately